samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorFrontDoorMessages_pt_BR.properties

```diff
@@ -41,7 +41,6 @@ anonymous_users=Usu\u00e1rios An\u00f4nimos
 select_action=A\u00e7\u00e3o
 entire_site=Todo o Site
 
-# bjones86 - OWL-1146

```

Comment:
I don't know if this is the best place to remove this line from a samigo bundle.

Reply:
Probably there is no 'best place' to remove this reference; it was an error getting into the code in the first place. But I don't think it would warrant it's own JIRA ticket either...

Reply:
Is this really a blocker? I can get rid of it, but I really don't think it's a big deal. There is zero risk associated with this change in my opinion.